src/Command/Redis/TFCALLASYNC.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis;
+
+class TFCALLASYNC extends TFCALL
```

Comment:
@shacharpash this is what I meant with [this comment](https://github.com/redis/redis-py/pull/2861#discussion_r1280260752) in redis-py. I think you two should align on an API